Toolbar customization/ReplaceToolbarItem/ReplaceToolbarItem/README.md

```diff
@@ -0,0 +1,119 @@
+# How to replace a built-in toolbar item with a custom application-level control in .NET MAUI PDF Viewer?
+This project demonstrates how to replace the existing built-in toolbar item with a custom application-level control in the index of the removed existing built-in toolbar item in .NET MAUI PDF Viewer. As an example, the built-in toolbar item we consider for replacing is the "MoreOptions" item, which includes "Outline" and "Print" as dropdown options on the top toolbar in both Android and iOS platforms and the custom application-level control is button.
+
+## Prerequisites
+1. A .NET MAUI project set up.
+2. The [Syncfusion.Maui.PdfViewer](https://www.nuget.org/packages/Syncfusion.Maui.PdfViewer) package is installed.
+
+## Steps
+
+### 1.Install Required NuGet Package
+To get started, create a new [Maui App](https://dotnet.microsoft.com/en-us/learn/maui/first-app-tutorial/create) and ensure the following package is installed in your .NET MAUI project:
+
+[Syncfusion.Maui.PdfViewer](https://www.nuget.org/packages/Syncfusion.Maui.PdfViewer)
+
+You can install this package using the NuGet Package Manager or the NuGet CLI.
+
+### 2. Initialize the PDF Viewer in XAML
+
+Start by adding the Syncfusion PDF Viewer control to your XAML file.
+
+**a. Add the Syncfusion namespace in your MainPage.xaml:**
+
+Define the XAML namespace to enable access to the PDF Viewer.
+
+**XAML:**
+
+```xaml
+xmlns:syncfusion="clr-namespace:Syncfusion.Maui.PdfViewer;assembly=Syncfusion.Maui.PdfViewer"
+```
+
+**b. Add the PDF Viewer control to your layout:**
+
+Initialize the SfPdfViewer in the XAML file. This will display the PDF Viewer in your app. You can load any PDF document into this Viewer and wire the DocumentLoaded event for the PdfViewer.
+
+**XAML:**
+
+```xaml
+     <Grid>
+        <syncfusion:SfPdfViewer x:Name="pdfViewer" ></syncfusion:SfPdfViewer>
+     </Grid>
+```
+
+**c. Load the PDF in the PDF Viewer control:**
+
+In your MainPage.xaml.cs, the PDF Viewer is initialized with a PDF document embedded in your resources.
+
+```csharp
+    Stream? stream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("ReplaceToolbarItem.Assets.PDF_Succinctly.pdf");
+    pdfViewer.LoadDocument(stream);
+```
+
+**d. Wire the document loaded event for the pdfViewer control**
+
+The event is triggered when the document loaded.
+
+**XAML:**
+
+```xaml
+    <syncfusion:SfPdfViewer x:Name="pdfViewer" DocumentLoaded="PdfViewerDocumentLoaded"></syncfusion:SfPdfViewer>
+```
+
+### 3. Replace the existing item with another item in the top toolbar in Android and IOS platform.
+
+In the wired document loaded event:
```

Comment:
Fix grammatical issues (e.g., “In the wired document loaded event” should be “In the DocumentLoaded event handler”).